Sources/Plasma/CoreLib/HeadSpin.cpp

```diff
@@ -298,6 +298,7 @@ class hsMinimizeClientGuard
 
 bool hsMessageBox_SuppressPrompts = false;
 
+#ifndef HS_BUILD_FOR_APPLE
```

Comment:
Might want to add a comment about this - it was a bit confusing for me when I hadn't seen the separate HeadSpin_Mac.mm yet.

Reply:
Out of scope for this PR, but as a general comment it would probably be worth splitting the Win32 code out into `HeadSpin_Win32.cpp` or something for consistency